external/nlohmann/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+add_library(nlohjson INTERFACE)
+target_include_directories(nlohjson INTERFACE ${sst-core_SOURCE_DIR}/../external)
```

Comment:
Is this going to be included downstream? If so, you will need to install it and use BUILD_INTERFACE/INSTALL_INTERFACE generator expressions.